website/webpack.config.js

```diff
@@ -39,24 +39,60 @@ module.exports = {
         test: /\.css$/,
         use: [{ loader: 'style-loader' }, { loader: 'css-loader' }],
       },
+      {
+        test: /\.(png|svg|jpg|gif)$/,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              publicPath: 'assets',
+              context: __dirname,
+              outputPath(_url, resourcePath, context) {
+                return path.relative(context, resourcePath);
+              },
+              name: '[name].[ext]',
+            },
+          },
+        ],
+      },
+      {
+        test: /\.webmanifest$/,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              publicPath: 'assets',
+              context: __dirname,
+              outputPath(_url, resourcePath, context) {
+                return path.relative(context, resourcePath);
+              },
+              name: '[name].[ext]',
+            },
+          },
+          {
+            loader: 'webmanifest-loader',
+          },
+        ],
+      },
     ],
   },
   performance: {
     hints: false,
   },
-  plugins: examples
-    .map(
+  plugins: [
+    ...examples.map(
       example =>
         new HTMLWebpackPlugin({
           template: path.join(__dirname, example, 'index.html'),
           filename: path.join(outputPath, example, 'index.html'),
           chunks: [example],
         })
-    )
-    .concat([
-      new CopyWebpackPlugin([
-        { from: path.join(__dirname, 'assets'), to: 'assets/' },
-        path.join(__dirname, '_redirects'),
```

Comment:
It looks like we forgot this line inside the change. The redirects don't work anymore.

> https://deploy-preview-2531--react-instantsearch.netlify.com/

Reply:
Put back in 816ec028ef46b7d54040d51f602e1f82da1f39a1.